hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -84,7 +84,7 @@ contracts.scheduleThrottleMaxGasLimit=5000000
 contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE
 contract.storageSlotPriceTiers=0til100M,2000til450M
 contracts.throttle.throttleByGas=true
-contracts.precompile.atomicCryptoTransfer.enabled=false
+contracts.precompile.atomicCryptoTransfer.enabled=true
```

Comment:
@lukelee-sl Maybe drop this and we can get the tests alone in

Reply:
dropped